### ISSUE TITLE
Bug: Render on deadline not working

### ISSUE THREAD

**Issue by romaincharoy** [quadproduction/OpenPype#573]
### Before posting a new ticket, have you looked through the documentation to find an answer?

Yes I have

### Have you looked through the existing tickets to find any related issues ?

Yes I have

### Author of the bug

Romain Charoy

### Version

3.15.12-quad.4.1

### What platform you are running OpenPype on?

Linux / Centos

### Current Behavior:

When attempting to launch a render from maya to deadline, there is an error on deadline

### Expected Behavior:

Full sequence rendering without error

### What type of bug is it ?

Happened only once in a particular configuration

### Which project / workfile / asset / ...

/prod/prod2/Projets/2022/TEST_OP_RLX_2022/Sequences/101/101_0010/work/light/test_op_rlx_101_0010_light_v006.ma

### Steps To Reproduce:

Open the described lighting scene in maya from the launcher.
Pyblish the renderlayer and workfile.
Once the job is on deadline you get the error.
Job ID  64ecd023181037062749fe8f for example

### Relevant log output:

```shell
023-08-29 10:55:20:  0: STDOUT: ERROR - Skipped: "submit_houdini_render_deadline" (No module named 'hou')
2023-08-29 10:55:20:  0: STDOUT: ERROR:pyblish.plugin:Skipped: "submit_houdini_render_deadline" (No module named 'hou')
2023-08-29 10:55:20:  0: STDOUT: ERROR - Skipped: "submit_nuke_deadline" (No module named 'nuke')
2023-08-29 10:55:20:  0: STDOUT: ERROR:pyblish.plugin:Skipped: "submit_nuke_deadline" (No module named 'nuke')
2023-08-29 10:55:20:  0: STDOUT: ERROR - Skipped: "submit_maya_remote_publish_deadline" (No module named 'maya')
2023-08-29 10:55:20:  0: STDOUT: ERROR:pyblish.plugin:Skipped: "submit_maya_remote_publish_deadline" (No module named 'maya')
2023-08-29 10:55:20:  0: STDOUT: ERROR - Skipped: "submit_maya_deadline" (No module named 'maya')
2023-08-29 10:55:20:  0: STDOUT: ERROR:pyblish.plugin:Skipped: "submit_maya_deadline" (No module named 'maya')
2023-08-29 10:55:20:  0: STDOUT: ERROR - Skipped: "submit_houdini_remote_publish" (No module named 'hou')
2023-08-29 10:55:20:  0: STDOUT: ERROR:pyblish.plugin:Skipped: "submit_houdini_remote_publish" (No module named 'hou')
2023-08-29 10:55:20:  0: STDOUT: ERROR - Skipped: "collect_deadline_server_from_instance" (No module named 'maya')
2023-08-29 10:55:20:  0: STDOUT: ERROR:pyblish.plugin:Skipped: "collect_deadline_server_from_instance" (No module named 'maya')
2023-08-29 10:55:20:  0: STDOUT: ERROR - Skipped: "submit_max_deadline" (No module named 'pymxs')
2023-08-29 10:55:20:  0: STDOUT: ERROR:pyblish.plugin:Skipped: "submit_max_deadline" (No module named 'pymxs')
2023-08-29 10:55:22:  0: STDOUT: ERROR - Skipped: "validate_arnold_camera_metadata" (No module named 'maya')
2023-08-29 10:55:22:  0: STDOUT: ERROR:pyblish.plugin:Skipped: "validate_arnold_camera_metadata" (No module named 'maya')

2023-08-29 10:55:24:  0: STDOUT: FileNotFoundError: [Errno 2] No such file or directory: '/prod/prod2/Projets/2022/TEST_OP_RLX_2022/INSTA_sept2023/0010/work/light/renders/maya/test_op_rlx_0010_workfileLight_v005/LIGHT/MONTRE_LAYER_RGBA_Light_Montre_Key'
2023-08-29 10:55:24:  0: STDOUT: ERROR:pyblish.plugin:Traceback (most recent call last):
2023-08-29 10:55:24:  0: STDOUT:   File "/prod/softprod/apps/openpype/LINUX/3.15.0/dependencies/pyblish/plugin.py", line 527, in __explicit_process
2023-08-29 10:55:24:  0: STDOUT:     runner(*args)
2023-08-29 10:55:24:  0: STDOUT:   File "/users_roaming/rcharoy/.local/share/openpype/3.15/openpype-v3.15.12-quad.4.0/openpype/modules/deadline/plugins/publish/validate_expected_and_rendered_files.py", line 30, in process
2023-08-29 10:55:24:  0: STDOUT:   File "/users_roaming/rcharoy/.local/share/openpype/3.15/openpype-v3.15.12-quad.4.0/openpype/modules/deadline/plugins/publish/validate_expected_and_rendered_files.py", line 191, in _get_existing_files
2023-08-29 10:55:24:  0: STDOUT: FileNotFoundError: [Errno 2] No such file or directory: '/prod/prod2/Projets/2022/TEST_OP_RLX_2022/INSTA_sept2023/0010/work/light/renders/maya/test_op_rlx_0010_workfileLight_v005/LIGHT/MONTRE_LAYER_RGBA_Light_Montre_Key'
2023-08-29 10:55:24:  0: STDOUT: Traceback (most recent call last):
2023-08-29 10:55:24:  0: STDOUT:   File "/prod/softprod/apps/openpype/LINUX/3.15.0/dependencies/pyblish/plugin.py", line 527, in __explicit_process
2023-08-29 10:55:24:  0: STDOUT:     runner(*args)
2023-08-29 10:55:24:  0: STDOUT:   File "<string>", line 30, in process
2023-08-29 10:55:24:  0: STDOUT:   File "<string>", line 191, in _get_existing_files
2023-08-29 10:55:24:  0: STDOUT: FileNotFoundError: [Errno 2] No such file or directory: '/prod/prod2/Projets/2022/TEST_OP_RLX_2022/INSTA_sept2023/0010/work/light/renders/maya/test_op_rlx_0010_workfileLight_v005/LIGHT/MONTRE_LAYER_RGBA_Light_Montre_Key'
```

**Comment by romaincharoy** [comment]
Hi @ccaillot 
understood. What is the next step with the first issue ? 
We tried to launch jobs with another watch (ID > 64f082a218103706274a9e12) and killing the publish job and all the dependencies. The job stays on "queued". 
Thank you

**Comment by ccaillot** [comment]
Hi @romaincharoy,

It's normal, as I said in my first message, you need to fix this node in yur scene (COSMO_40_Mxxxxxx_xxxx_01_:metal_multiFinition_acier_depoli:file_metal_multiFinition_input2_0).
It still exists yet in your scene, while this node will be there the render will be blocked.


![image](https://github.com/quadproduction/OpenPype/assets/134272390/3cd0ae1d-ba62-4a18-b089-5770a4a3197d)

**Comment by ccaillot** [comment]
by executing this code in your script editor you can check all dependencies your scene are calling during the deadline processing.

If you see a path which is equal to None, this will block your render.... but there is a workaround in deadline which consist in your job to delete the 'blank path' in Job Properties > Dependencies >Assets Tab > Remove on Right Top (While blank path is selected) (cf. screenshot).

```
dependencies = []
for node in cmds.ls(type="file"):
    path = cmds.getAttr("{}.{}".format(node, "fileTextureName"))
    if path not in dependencies:
        print(node, path)
        dependencies.append(path)

    for node in cmds.ls(type="AlembicNode"):
        path = cmds.getAttr("{}.{}".format(node, "abc_File"))
        if path not in dependencies:
            print(node, path)
            dependencies.append(path)

```

PS: for blank path like the node I've pointed before, you can break the connection and use a black constant, it will make the same job.

![Image](https://github.com/quadproduction/OpenPype/assets/134272390/11507ac1-63f4-4916-9936-503c9a9fc74f)

**Comment by romaincharoy** [comment]
ok thank you !
